src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserAccountResourceIntTest.kt

```diff
@@ -170,7 +170,7 @@ class UserAccountResourceIntTest : IntegrationTestBase() {
         )
 
       webTestClient.get().uri("/users/TESTUSER2")
-        .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_ACCESS_ROLES_ADMIN")))
+        .headers(setAuthorisation(roles = listOf("ROLE_MAINTAIN_ACCESS_ROLES")))
```

Comment:
Then I suppose add an additional test for ROLE_MANAGE_NOMIS_USER_ACCOUNT rather than replacing ROLE_MAINTAIN_ACCESS_ROLES_ADMIN